system_metrics_collector/src/system_metrics_collector/linux_process_cpu_measurement_node.hpp

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SYSTEM_METRICS_COLLECTOR__LINUX_PROCESS_CPU_MEASUREMENT_NODE_HPP_
+#define SYSTEM_METRICS_COLLECTOR__LINUX_PROCESS_CPU_MEASUREMENT_NODE_HPP_
+
+#include <chrono>
+#include <cmath>
+#include <string>
+#include <tuple>
+
+#include "../../src/system_metrics_collector/periodic_measurement_node.hpp"
```

Comment:
Why the relative include paths? These should be `#include "system_metrics_collector/foo.hpp"`

Reply:
Okay, I guess I can start doing that here in this new file. The other `.cpp` files have been doing it like this...

Reply:
We don't have an include directory for the first release....

Reply:
never too late to fix an error - sorry you're the one who got caught with it heh

Reply:
i could be convinced that it should be a separate PR to fix the relative include paths, as long as we make sure to actually do it. i'd personally prefer to just take care of it now that we've noticed, but it's up to you

Reply:
@dabonnie you can `-I` / `include_directories` / `target_include_directories` any directory. The CMakeLists should be taking care of this for the targets that need to include these headers, not the source files, this pattern is brittle because it required refactoring if the directory structure ever changes

Reply:
I've updated the paths here to be minimally relative.

We should also open an issue tracking the to-be separate PR for fixing the relative include paths.